content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -49,7 +49,7 @@ To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/
       * `--feature-gates="IPv6DualStack=true"`
    * kube-controller-manager:
       * `--feature-gates="IPv6DualStack=true"`
-      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--cluster-cidr=10.244.0.0/16,fc00::/48`
+      * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
       * `--service-cluster-ip-range=<IPv4 CIDR>,<IPv6 CIDR>` eg. `--service-cluster-ip-range=10.0.0.0/16,fd00::/108`
```

Comment:
@bridgetkromhout I'd love to merge this as is _but_ I have a concern. Does this line also need to change?

Reply:
In the associated issue, the issue creator pointed out that the `fc00::/48` segment was misleading. I kept the IPv4 example but I moved it into the notes section below, and I clarified the IPv6 example so it would no longer be misleading.

Reply:
@markzzzsmith have you got time to comment here? I think that `fd00::/108` is also problematic because
>  the 40 bits after the first 8 bits of fd00::/8 are required to be a random number that is likely globally unique

and so, although this PR as it stands is a partial fix, there is more to do.